backend/models/boards.js

```diff
@@ -133,6 +136,14 @@ async function userInBoard(boardId, userId) {
 async function addDevToBoard(boardId, devIds) {
   const db = await dbHandler;
 
+  const userCheck = await db.all('SELECT id FROM Users');
+
+  for (let i = 0; i < devIds.length; i += 1) {
+    if (!userCheck.some((member) => member.id === devIds[i])) {
```

Comment:
Add lang guro ug TODO nganhi nga i optimize ni siya. what if in the future 1000000 na kabuok atong users. so dapat i query tanan. better guro nga i select nalang nato individually. mas daghan tag queries pero I think mas optimized siya. DLi ra siya urgent though kay gamay raman sad users nato

Reply:
in that case, pwede ra jud ug i remove ni siya. nagpahimo man to si justin ug user/exists nga endpoint, and nahimo na man, so i guess better if kana na lang ang gamiton?

Reply:
hmmm ngano gae nagpahimo siya ato nga endpoint?

Reply:
pero yeah pwede ra icall to nga function

Reply:
same reason ngano nag add ko anang userCheck - para icheck if existing ang user. pero ang user/exists kay mag base sa email

Reply:
ay ka g ko sa purpose pero ngano gicheck na sa frontend?

Reply:
wala sad koy sure ngano @justinpo